tensorflow/tools/pip_package/BUILD

```diff
@@ -131,6 +131,7 @@ filegroup(
     ] + if_mkl([
         "//third_party/mkl:LICENSE",
         "@mkl//:LICENSE",
+        "@mkl_dnn//:LICENSE",
```

Comment:
Since this package is excluded from the pip package, we can remove its license.
Only need its license if we have it in the pip package.

Reply:
Done.